test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -6172,6 +6172,15 @@ def forward(self, input):
         model = Log1p()
         self.run_test(model, x)
 
+    @skipIfUnsupportedMinOpsetVersion(9)
+    def test_log10(self):
+        class Log10(torch.nn.Module):
+            def forward(self, input):
+                return torch.log1p(input)
+        x = torch.rand(2, 3, 4)
+        model = Log10()
+        self.run_test(model, x)
```

Comment:
please in-line the Log10():

`self.run_test(Log10(), x)`

Reply:
done